opencti-platform/opencti-front/tests_e2e/settings/parameters.spec.ts

```diff
@@ -0,0 +1,48 @@
+import { test } from '@playwright/test';
+import { expect } from '../fixtures/baseFixtures';
+import { LeftBarPage } from '../model/menu/leftBar.pageModel';
+
+test('Check EE activation', async ({ page }) => {
+  const leftBarPage = new LeftBarPage(page);
+
+  await page.goto('/');
+  await page.getByTestId('ChevronRightIcon').click();
+  await leftBarPage.clickOnMenu('Settings', 'Parameters');
+
+  await page.getByRole('button', { name: 'Enable Enterprise Edition' }).click();
+  await expect(page.getByRole('heading', { name: 'OpenCTI Enterprise Edition (' })).toBeVisible();
+  await page.getByLabel('I have read and agree to the').check();
+  await page.getByRole('button', { name: 'Enable' }).click();
+  expect(page.locator('div').filter({ hasText: /^Enterprise$/ })).toBeVisible();
```

Comment:
Why not using the dedicated locator `getBytext`? https://playwright.dev/docs/locators#locate-by-text

Reply:
The playwright generator gave me this, I trusted him :)

Reply:
You're better than Playwright, gg.